.gitignore

```diff
@@ -41,3 +41,4 @@ __pycache__
 /tests/unit/bintest/
 /tests/unit/objtest/
 /tests/ystests
+/.vs
```

Comment:
Guess you are using Visual Studio as editor, I have no problem with this in .gitignore but then we would need to add /.vscode and other temp folders as well